tests/Jackalope/ObjectManagerTest.php

```diff
@@ -55,12 +55,11 @@ public function testRegisterUuid()
         $this->assertInstanceOf(Node::class, $node);
     }
 
-    /**
-     * @expectedException \RuntimeException
-     * @expectedExceptionMessage Object path for UUID "1234" has already been registered to "/path/to/this"
-     */
     public function testRegisterUuidAlreadyMapped()
     {
+        $this->expectException(\RuntimeException::class);
+        $this->expectExceptionMessage('Object path for UUID "1234" has already been registered to "/path/to/this"');
```

Comment:
please put this after the first `registerUuid` call. the upside of having the expect exceptions in code is that they can be at the precisely correct place.

Reply:
done